test/unit/test_actions.py

```diff
@@ -452,7 +452,7 @@ def test_record_state(self, xpatch):
         xpatch.system.socket_data(test_state_data.encode())
         output = self.invoke_action('record-state',
                                     state_name='test_state',
-                                    format="c-style",
+                                    format="nso-c-style",
                                     overwrite=False,
```

Comment:
Doesn't this mean that the default format for the record-state action is nso-c-style?

Reply:
Not sure I understand, this is just a unit test that stopped working after the change. The problem was introduced in 878c03c4 where state format enums were renamed (`xml` -> `nso-xml` in particular), but the code was checking for `xml` and assumed C-style if that check failed.

Reply:
You're right. My mistake.